### PR TITLE
feat: runtime metrics UI

### PR DESCRIPTION
**Pull request by 4nalog**
# TL;DR

This PR adds logic for allowing runtime task metrics to me displayed in the timeline chart component

To run:
- setup the project by cloning
- run `yarn install` and `yarn start`
- navigate to a specific workflow and check out the timeline view

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

This feature:
- add logic to fetch task level runtime metrics in the timeline view using the new workflow execution metrics api
- destructures the data returned from the workflow execution metrics api to get values from the corresponding phases in detail based on the operation id
- displays those as a chart on the timeline

## Tracking Issue

fixes

### CODE REVIEW

**Comment by tsheiner** [comment]
@4nalog here I am

**Comment by jsonporter** [comment]
Bugs per our sync with @hamersaw:
 - Diff on seconds first
 - append nanos (if present) to result of seconds diff.

**Comment by tsheiner** [comment]
@4nalog I see two issues:
- the way the tooltip animates is not OK; it kind of flys in from somewhere. Please just fade it up in .5ms at mouse position.
- I think we are still drawing the lines partitioning the bar; please suppress.

**Review by jsonporter — APPROVED** [review]
